_posts/2022-06-06-roadmap-to-successful-data-engineer.md

```diff
@@ -65,7 +65,7 @@ The most common skill that all data engineers should possess is obviously SQL, b
 
 A Java virtual machine (JVM) is a virtual machine that enables a computer to run Java programs as well as programs written in other languages that are also compiled to Java bytecode. The JVM is detailed by a specification that formally describes what is required in a JVM implementation. Let us see the advantages of JVM.
 
-- Platform independence: Java is meant for “write once, run anywhere”. Native languages like C/C++ compiled to match the specific platform, while Java code is compiled into bytecode. JVMs can interpret bytecode instructions into the native language for the specific platform.
```

Comment:
I'd rather keep a noun here, like "independence", because all bullets are "titled" with a noun. It's a technique that I like to call "bullet parallelism" in technical writing - it promotes consistency between bullets in a list.